fluent-package/test-install-in-docker.sh

```diff
@@ -14,10 +14,11 @@ set -e
 
 function setup_apt_user()
 {
+    export DEBIAN_FRONTEND=noninteractive
```

Comment:
Is this for safety?
Should we do this around the following code instead of in the function if it is not limited to user setup?

https://github.com/fluent/fluent-package-builder/blob/7991bbb68ccf36439e200b208b2d2d8cb02d7b63/fluent-package/test-install-in-docker.sh#L92-L93

Reply:
Surely it may be better to do so.
Thanks!